sdk/how-to-guides/manage-storage.md

```diff
@@ -0,0 +1,232 @@
+> See first: [Juju | How to Manage Storage](https://juju.is/docs/juju/manage-storage)
+
+**Contents:**
+
+- [Implement storage management](#heading--implement-storage-management)
```

Comment:
Why "storage management"? Actually I think I understand why, but it's clunky. Thinking...